IFC4x3/Properties/s/SleeperArrangement_24pyO2n$n5EOpil0HhjmwF/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SleeperArrangement_24pyO2nn5EOpil0HhjmwF" Name="SleeperArrangement" UniqueId="84cfc602-c7fc-4539-8cec-bc046bb70e8f" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Could use `ConstructionMethod` again in my view.

Reply:
Not the same property, Here it defines if the rail joint is constructed on a twinsleeper or between sleepers.